CMakeLists.txt

```diff
@@ -12,6 +16,8 @@ option(S2N_NO_PQ "Disables all Post Quantum Crypto code. You likely want this
 for older compilers or uncommon platforms." OFF)
 option(S2N_NO_PQ_ASM "Turns off the ASM for PQ Crypto even if it's available for the toolchain.
 You likely want this on older compilers." OFF)
+option(SEARCH_LIBCRYPTO "Set this if you want to let S2N search libcrypto for you,
+otherwise a crypto target need to be defined." ON)
```

Comment:
Nit
```suggestion
option(SEARCH_LIBCRYPTO "Set this if you want to let S2N search libcrypto for you,
otherwise a crypto target needs to be defined." ON)
```